cfme/tests/containers/test_autotagging_providers.py

```diff
@@ -0,0 +1,199 @@
+from collections import namedtuple
+import pytest
+from random import choice
+import time
+
+import fauxfactory
+from cfme.configure.configuration.region_settings import MapTags
+from cfme.containers.provider import ContainersProvider
+from cfme.containers.node import Node
+from cfme.containers.replicator import Replicator
+from cfme.containers.image import Image
+from cfme.containers.pod import Pod
+from cfme.containers.project import Project
+from cfme.containers.route import Route
+
+from utils import testgen
+from utils.version import current_version
+from utils.wait import wait_for
+
+
+pytestmark = [
+    pytest.mark.uncollectif(lambda provider: current_version() < "5.8"),
+    pytest.mark.usefixtures('setup_provider_modscope'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate([ContainersProvider], scope='module')
+
+TEST_OBJECTS = (Pod, Route, Node, Replicator, Project)
+
+VALUES_TUPLE = ('value_' + fauxfactory.gen_alpha(5).lower(),
+                'value_' + fauxfactory.gen_alphanumeric(5),
+                'value_' + fauxfactory.gen_alpha(3).lower() + fauxfactory.gen_alphanumeric(3))
+
+
+def generate_values():
+    values = []
+    for value in VALUES_TUPLE:
+        value_string = value
+        values.append(value_string)
+    return values
```

Comment:
**REQUIRED**
What's the purpose of this function? It just converts `VALUES_TUPLE` tuple to a list.